lib/rules/block-closing-brace-newline-after/index.js

```diff
@@ -99,6 +99,22 @@ const rule = function(expectation, options) {
         index: -1,
         lineCheckStr: blockString(statement),
         err: msg => {
+          if (context.fix) {
+            if (expectation.indexOf("always") === 0) {
+              const index = nodeToCheck.raws.before.search(/\r?\n/);
+              if (index >= 0) {
+                nodeToCheck.raws.before = nodeToCheck.raws.before.slice(index);
+              } else {
+                nodeToCheck.raws.before =
+                  context.newline + nodeToCheck.raws.before;
+              }
+              return;
+            } else if (expectation.indexOf("never") === 0) {
```

Comment:
Way not just use:
```js
if (expectation === "never") {
```

Reply:
@gucong3000 It matches `never-single-line`, `never-multi-line`.